### DISCUSSION TITLE
How to change default language?

### DISCUSSION THREAD

**Discussion by tommydurante** [jekyll/jekyll#9719] [Q&A]
Hello, i'm very new to the world of jekyll and im trying to set up Spanish as the language but i can't find the file that should go in _layouts folder. 
![image](https://github.com/user-attachments/assets/51d3c1a0-b0c6-46a0-aded-4e0a0e9948f9)

Can anyone explain me please? thanks, sorry if my english is bad!

**Comment by ashmaroli** [comment]
Hello @tommydurante, Welcome to Jekyll!!

Jekyll doesn't have a built-in language-management functionality. Therefore, it gives theme-authors full control over how a particular theme handles multiple languages.

The recommended solution is to consult the theme's documentation and if you are unable to understand the process in spite of that, the alternative solution would be to ask for support directly from the theme-author / maintainer(s).
